src/core/client/ui/components/v3/ComponentLink/ComponentLink.tsx

```diff
@@ -0,0 +1,16 @@
+import cn from "classnames";
+import React, { FunctionComponent } from "react";
+
+import styles from "./ComponentLink.css";
+
+const ComponentLink: FunctionComponent<{
+  href: string;
+  children: React.Component | Element | React.ReactNode;
+  className?: string;
+}> = ({ href, children, className }) => (
+  <a href={href} className={cn(styles.root, className)}>
+    {children}
+  </a>
+);
+
+export default ComponentLink;
```

Comment:
When do we need this component where `<TextLink>` and regular `<a>` won't suffice?

Reply:
Replaced and fixed